_data/topnav.yml

```
@@ -3,6 +3,8 @@ topnav:
   items:
     - title: Blog
       external_url: https://aka.ms/azsdk/blog
+    - title: Roadmap
+      url: /roadmap
```

I think this needs to end in a slash to fix the broken link check.
```suggestion
      url: /roadmap/
```